common/src/macOSMain/kotlin/com/halcyonmobile/multiplatformplayground/shared/CoroutineViewModel.kt

```diff
@@ -0,0 +1,28 @@
+package com.halcyonmobile.multiplatformplayground.shared
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.cancel
+
+actual abstract class CoroutineViewModel {
```

Comment:
We should try to move these into a common `appleMain` sourceSet. (See [dependsOn](https://kotlinlang.org/docs/reference/mpp-share-on-platforms.html#configure-the-hierarchical-structure-manually))

I can play around with it.